sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_region.c

```diff
@@ -38,11 +39,24 @@ int gnrc_lorawan_set_dr(gnrc_lorawan_t *mac, uint8_t datarate)
     return 0;
 }
 
+#if (IS_ACTIVE(CONFIG_LORAMAC_REGION_EU_868))
 uint8_t gnrc_lorawan_rx1_get_dr_offset(uint8_t dr_up, uint8_t dr_offset)
 {
     return (dr_up > dr_offset) ? (dr_up - dr_offset) : 0;
 }
 
+#elif (IS_ACTIVE(CONFIG_LORAMAC_REGION_IN_865))
+uint8_t gnrc_lorawan_rx1_get_dr_offset(uint8_t dr_up, uint8_t dr_offset)
+{
+    if (dr_offset < 6) {
+        return (dr_up <= dr_offset) ? 0 : (dr_up - dr_offset) > 5 ? 5 : (dr_up - dr_offset);
+    }
+    else {
+        return (dr_offset + dr_up) > 10 ? 5 : ((dr_offset - 5) + dr_up);
+    }
```

Comment:
Last nit pick.
I find this a little bit hard to follow.  Maybe something like this?

```suggestion
    int dr_eff = dr_offset > 5 ? 5 - dr_offset : dr_offset;
    return MIN(5, MAX(0, dr_up - dr_eff));
```

Given:
```
#define MIN(a, b) ((a) < (b) ? (a) : (b))
#define MAX(a, b) ((a) > (b) ? (a) : (b))
```

Reply:
Makes sense :+1: